LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014 ClusterHQ
+   Copyright 2014, 2015 ClusterHQ
```

Comment:
I'll leave this - we can change the company name everywhere at once (at the top of source files for example) when necessary.